src/ui/templates/results/resultsheader.hbs

```diff
@@ -30,8 +30,16 @@
             <span class="yxt-ResultsHeader-filterLabelColon">:</span>
           </div>
         {{/if}}
-        {{#each this.displayValues}}
-          <div class="yxt-ResultsHeader-filterValue">{{this}}{{#unless @last}}, {{/unless}}</div>
+        {{#each filterDataArray}}
+          {{#if removable}}
+            <div class="yxt-ResultsHeader-removableFilterTag js-yxt-ResultsHeader-removableFilterTag"
+              data-filter-id={{dataFilterId}}>
+              <span class="yxt-ResultsHeader-removableFilterValue">{{displayValue}}</span>
+              <span class="yxt-ResultsHeader-removableFilterX">&times;</span>
+            </div>
+          {{else}}
+            <div class="yxt-ResultsHeader-filterValue">{{displayValue}}{{#unless @last}}, {{/unless}}</div>
```

Comment:
Do we want spans in this case as well? Could be good to add now in case we want to target that text with styling at some point.

Reply:
sure! I love making consulting happy no cap